content/en/docs/collector/custom-collector.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Building your own Collector instance
+weight: 30
```

Comment:
In which position do we want this page to appear in the subpage list? The custom auth page also has weight 30, so these two will be ordered relative to each other based on the page titles. Is that what we want?

Reply:
In my opinion the development of any Collector's component starts with having an environment to properly build your component and ends with a Collector's distribution that has the component in it.

So I would expect this document to show up as the first, and then everything else follows

Reply:
Weight updated (I'll adjust weights across files later).